src/App.js

```diff
@@ -0,0 +1,254 @@
+
+const Instructions = (props) => {
+  return (
+    <div className='image-comparison-tool'>
+      <h2>Upload a CSV file to start comparing images</h2>
+      <p>This tool will display a side-by-side snapshot of two URLs from CSV.</p>
+      <h3>How to use</h3>
+      <ol>
+        <li>Prepare your CSV file with the following columns: <code>oldUrl</code>, <code>newUrl</code>, <code>status</code>.</li>
+        <li>Upload your CSV file.</li>
+        <li>You can view the results by clicking on the <b>Previous</b> and <b>Next</b> buttons.</li>
+        <li>To add a status or test result, click on <b>Accept</b> button to mark the data as passed. Otherwise, click on the <b>Reject</b> button.</li>
+        <li>Once you are done, download the CSV file.</li>
+      </ol>
+      
+    </div>
+  )
+}
+const Navigation = (props) => {
+  return (
+    <div className="navigation">
+      <button onClick={props.onPrev}>{'\u2B05'} Previous</button>
+      <button onClick={props.onModalClose}>Close</button>
+      <button onClick={props.onNext}>Next {'\u27A1'}</button>    
+    </div>
+  )
+}
+
+const Download = (props) => {
+  return (
+    <div className="download">
+      <button id="downloadBtn" onClick={props.onDownload} value="download">Download</button>
+    </div>
+  )
+}
+
+const Preview = (props) => {
+  return (
+    <div id="preview-all">
+      <div className="column">
+        <p>
+          <strong>PAGE {props.activeData + 1}:</strong>
+          <a href={props.oldUrl}> {props.oldUrl}</a>
+        </p>
+        <iframe title="preview-0" className="preview" src={'https://image.thum.io/get/width/1024/fullpage/allowJPG' + props.oldUrl} width="100%" height="500" />
+      </div>
+      <div className="column">
+        <p>
+          <strong>PAGE {props.activeData + 1}:</strong>
+          <a href={props.newUrl}> {props.newUrl}</a>
+        </p>
+        <iframe title="preview-1" className="preview" src={'https://image.thum.io/get/width/1024/fullpage/allowJPG' + props.newUrl} width="100%" height="500" />
+      </div>
+    </div>
+  )
+}
+
+const Modal = (props) => {
+  return (
+      <div>
+        <div id="status">
+          <span id="status-text"></span>
+          <p>
+            <strong>Status:</strong> {props.status}
+          </p>
+        </div>
+        <div className="action">
+          <button className="accept" onClick={props.onAccept}>Accept</button>
+          <button className="reject" onClick={props.onReject}>Reject</button>
+        </div>
+        <Preview {...props}></Preview>
+        <div className="skip">
+          <p>To skip to the item without status: </p>
+          <button className="skip-button" onClick={props.onSkip}>Skip</button>
+        </div>
+        <Download 
+          onDownload={props.onDownload}>
+        </Download>
+        <Navigation {...props}></Navigation>
+      </div>
+  );
+}
+
+function App() {
+  const [parsedData, setParsedData] = useState([]);
+  const [newData, setNewData] = useState([]);
```

Comment:
FYI if you do decide to move `newData` down into the `Modal` component, you can optionally just "copy" the initial state from the `parsedData` state!

`const [newData, setNewData] = useState([...parsedData])`

Reply:
Done in this [file](https://github.com/Automattic/compare-snapshot-tool/pull/1/files#diff-c80a14057369079bbb58db6e6af9b9b1a67a4126ea2d5b2cea658c5cc5b80326R24). 😄